README.md

```diff
@@ -359,11 +359,11 @@ testIframe( testName, fileName,
   } );
 ```
 
-This loads a page, constructing a url with fileName `"./data/" + fileName`.
+This loads a page, constructing a URL with fileName `"./data/" + fileName`.
 The iframed page determines when the callback occurs in the test by
 including the "/test/data/iframeTest.js" script and calling
 `startIframeTest( [ additional args ] )` when appropriate. Often this
-will be after either document ready or `window.onload` fires.
+will be after either the document is ready or `window.onload` fires.
```

Comment:
`document ready` is a jQuery term which means a concrete thing, `the document is ready` doesn't seem to carry a similar meaning.